packages/lesswrong/components/ea-forum/onboarding/EAOnboardingStage.tsx

```diff
@@ -102,8 +109,62 @@ const styles = (theme: ThemeType) => ({
   spinner: {
     height: "20px !important",
   },
+  closeButtonOuter: {
+    position: "absolute",
+    top: 16,
+    right: 24,
+  },
+  close: {
+    cursor: "pointer",
+  },
+  logoutDialog: {
+    zIndex: theme.zIndexes.confirmLogoutModal,
+    minWidth: 300,
+  },
+  logoutDialogTitle: {
+    marginTop: 0,
+  },
+  logoutDialogText: {
+    marginTop: 0,
+    marginBottom: 2,
+  },
+  logoutDialogActions: {
+    marginRight: 16,
+    marginBottom: 16,
+  },
 });
 
+const LogoutConfirmationDialog = (
+  {open, onClose, classes}:
+  {open: boolean, onClose: () => void, classes: ClassesType<typeof styles>},
+) => {
+  const currentUser = useCurrentUser();
+  
+  const {EAButton, LWDialog, Typography} = Components;
+  return <LWDialog open={open} onClose={onClose} className={classes.logoutDialog}>
+    <DialogTitle disableTypography>
+      <Typography variant="display1" className={classes.logoutDialogTitle}>
+        Confirm Logout
```

Comment:
```suggestion
        Confirm logout
```
(To make the general pattern, but feel free to override if it looks off)